backend/infrahub/actions/tasks.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import annotations
+
+from infrahub_sdk.graphql import Mutation, Query
+from infrahub_sdk.node import InfrahubNode
+from prefect import flow
+from prefect.client.orchestration import get_client
+
+from infrahub.context import InfrahubContext  # noqa: TC001  needed for prefect flow
+from infrahub.core.constants import InfrahubKind
+from infrahub.core.registry import registry
+from infrahub.services import InfrahubServices  # noqa: TC001  needed for prefect flow
+from infrahub.trigger.models import TriggerType
+from infrahub.trigger.setup import setup_triggers
+from infrahub.workflows.catalogue import ACTION_RUN_GENERATOR
+from infrahub.workflows.utils import add_tags
+
+from .models import ActionTriggerRuleTriggerDefinition, EventGroupMember
+
+
+@flow(
+    name="add_node_to_group",
```

Comment:
We should agree on a standard but most flows are using a dash notation instead of underscore today, also I think it would make sense to add a prefix but happy to discuss that
```suggestion
    name="action-add-node-to-group",
```

Reply:
I can add a prefix and a discussion would be good. As we talked about potentially exposing these as GraphQL enums in the future (for some filter view) I mentioned that it could make more sense to use underscores instead of dashes, as I don't think dashes are supported by GraphQL.